### ISSUE TITLE
Typo in flow for version 1.4.0

### ISSUE THREAD

**Issue by ggravarr** [react-native-community/react-native-device-info#613]
## Summary

I think there's a typo in the flow declaration for the new getPowerState method.

|             |     |
| ----------- | --- |
| Version     | 1.4.0   |
| Affected OS | MacOS   |
| OS Version  | 10.13.6   |

## Current behavior
I get the following when checking flow on version 1.4.0

```
Error -- node_modules/react-native-device-info/deviceinfo.js.flow:47:32

Cannot resolve name object.

     44│   getTotalDiskCapacity: () => number,
     45│   getFreeDiskStorage: () => number,
     46│   getBatteryLevel: () => Promise<number>,
     47│   getPowerState: () => Promise<object>,
     48│   isBatteryCharging:() => Promise<boolean>,
     49│   isLandscape: () => boolean,
     50│   isAirPlaneMode: () => Promise<boolean>,
```

## Expected behavior

I imagine it was meant to be `Object` instead of `object`

**Comment by mikehardy** [comment]
Nice! I imagine there will be an n.n.n+1 release in just a moment while I track this down :-), thanks for reporting it

**Comment by mikehardy** [comment]
@ggravarr can you try master right now (I just committed the capitalization fix) to verify it works for you? If you confirm I release 1.4.1 immediately after, then I'll log a bug to install some CI so this doesn't happen again. I use Typescript not Flow so this just didn't come up in my testing but that's no excuse for a type error!

**Comment by ggravarr** [comment]
Yep, if I use master now it's fine.

**Comment by mikehardy** [comment]
I cross-checked myself, in the example I had to do these to check things:

`npm install --save-dev babel-cli babel-preset-flow flow-bin@^0.92.0`

(alternatively install flow current and edit .flowconfig to accept that version)

edit my babel presets to:
```  
presets: ['module:metro-react-native-babel-preset', 'flow'],
```
then I could `npx flow`

The example doesn't flow-check correctly but it doesn't throw an error on the definitions themselves.

I published 1.4.1!

**Comment by ggravarr** [comment]
Excellent, 1.4.1 works fine. Thanks.
